addon/routes/project/new.js

```diff
@@ -1,8 +1,8 @@
-import Route from "@ember/routing/route";
+import FormRoute from "./form";
 import { inject as service } from "@ember/service";
 import ConstructionProject from "ember-ebau-gwr/models/construction-project";
 
-export default class ProjectNewRoute extends Route {
+export default class ProjectNewRoute extends FormRoute {
```

Comment:
From where is the FormRoute?

Reply:
`import FormRoute from "./form";`

Reply:
Ah for ember the exported name doesn't have to be the same? e.g. `ProjectFormRoute`?